lib/features/authentication/data/datasources/authentication_local_data_source.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:convert';
+
+import 'package:coffeecard/features/authentication/data/models/authenticated_user_model.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:logger/logger.dart';
+
+class AuthenticationLocalDataSource {
```

Comment:
Don't refactor and add new features at the same time

Reply:
I disagree. Refactoring is part of implementing new features: leave the code prettier than when you found it.